src/app/modules/shared/components/search-panel/search-panel.component.html

```diff
@@ -1,4 +1,4 @@
-<div>
-  <input type="text">
-  <button>Search</button>
-</div>
+<form class="search-form">
+  <input #input class="search-form__input" type="text" [placeholder]="placeholder" [value]="inputValue | async" (input)="setInputValue($event)">
```

Comment:
Do line wrapping!
<input #input
         class="search-form__input"
         type="text" 
         [placeholder]="placeholder" 
         [value]="inputValue | async" 
         (input)="setInputValue($event)">

Reply:
And I have expected to see banana in the box 
Bind [()] input field with component property

Reply:
But you good, You have done it through rxjs

Reply:
line wrapping added in 5be0a87